### DISCUSSION TITLE
Arc Browser Support

### DISCUSSION THREAD

**Discussion by samaluk** [ttscoff/bunch-docs#299] [Q&A]
Hello, is there any intention to support the Arc Browser for opening urls?

**Comment by ttscoff** [comment]
Sure. Now that it it's your default browser, you can just not at a prefix at all…

Thanks,
Brett

**Reply by samaluk** [reply]
The thing is I use https://sindresorhus.com/velja for choosing my browser, It would be nice to be able to choose the Arc browser in bunch as well as choosing a space inside Arc

**Reply by ttscoff** [reply]
The current beta has support for an arc: prefix. I don't know if scripting spaces in the way I do with Chrome profiles will be feasible or not.
